server/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -185,8 +185,13 @@ private DivTag renderQuestionSummary(
                 .withClasses("font-semibold"))
             .withClasses("pr-2");
 
-    // Add existing answer.
-    if (data.isAnswered()) {
+    // When applicant info is pre-populated by TI entry, the question is not
+    // considered "answered" but we want the answers to show on the review screen
+    String defaultAnswerString = data.applicantQuestion().getQuestion().getDefaultAnswerString();
+    boolean haveAnswerText =
+        !data.answerText().isBlank() && !data.answerText().equals(defaultAnswerString);
```

Comment:
How is `-` handled for a11y?

Reply:
The screen reader doesn't say anything and it doesn't show up on the review screen, even though right now `-` is the answer string for almost every question that doesn't have an answer. I'm not clear on why `-` was set as the default and not `""`, but I left it since that's how it's been working and it doesn't seem to cause an issue.

I think you're right about that edge case not being an issue, Nick. I just tested it and it's impossible to enter `-` for dob, email, or phone. Answering a name question requires you answer first and last name fields, so it's impossible for it to be just `-`.